.env

```diff
@@ -0,0 +1 @@
+OPENAI_API_KEY=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
```

Comment:
⚠️⚠️⚠️ You published your API key, you must revoke it

Reply:
revoked